core/src/test/kotlin/org/eln2/sim/electrical/mna/CircuitTest.kt

```diff
@@ -106,14 +108,14 @@ internal class CircuitTest {
         r2.connect(0, ts.r1, 0)
         r2.connect(1, ts.r1, 1)
 
-        ts.c.step(0.5)
+        assert(ts.c.step(0.5))
```

Comment:
what does this assert do? It's not extremely clear

Reply:
`step` returns whether or not it succeeds; the most typical reason for it failing is encountering a singular matrix. If we're assuming that's not going to happen, asserting this is a good way to catch errors earlier (before the numerical asserts trigger).

Reply:
Ah, I see. Nice.